.ruby-version

```diff
@@ -0,0 +1 @@
+1.9.3-p551
```

Comment:
So is there any code in this repo that actually runs anywhere, apart from the specs? If not then this is just enforcing the ruby version for the specs, so we can make it a bit more modern...